### PR TITLE
Staff homepage hr card

### PR DESCRIPTION
**Pull request by bexar**
Adds HR card to homepage for staff affiliation only.

### CODE REVIEW

**Comment by klpittman** [comment]
screen shot 2017-07-10 at 11 50 35 am

Doesn't seem to be working. Am I missing something?

**Comment by bexar** [comment]
@klpittman add 'workday_account_card' to enabled feature list in settings.py

**Comment by bexar** [comment]
@klpittman yes, it should be there eventually but weren't we hiding it for student employees until workday release?

**Comment by klpittman** [comment]
You're right, I had forgotten that we decided to hide it from Student staff because it doesn't make as much sense as designed for them.

File: myuw/dao/affiliation.py [hunk 1]
```diff
@@ -61,6 +61,7 @@ def get_all_affiliations(request):
             "undergrad": is_undergrad_student(),
             "student": is_student(),
             "pce": is_pce_student(),
+            "staff_employee": is_staff_employee(),
```
Comment (fanglinfang) [inline myuw/dao/affiliation.py]: You may add it in affiliation_prefetch

File: myuw/static/js/landing.js [hunk 2]
```diff
@@ -41,24 +41,31 @@ var Landing = {
...
-            InternationalStuCard,
-            SummaryScheduleCard,
-            VisualScheduleCard,
-            TextbookCard,
-            FutureSummaryScheduleCard,
-            FutureQuarterCard1,
-            SummerRegStatusCard1
-        ];
+        var desktop_body_cards;
+        if(window.user.staff_employee) {
```
Comment (fanglinfang) [inline myuw/static/js/landing.js]: What if it is a staff employee and also taking a course?

Comment (klpittman) [inline myuw/static/js/landing.js]: Good question. In that case, we should default to the view typical Student would see. Visual schedule on the homepage. Workday card on the Accounts page. 

@bexar feel free to chime here if you think differently or I'm mistaken.

File: myuw/dao/affiliation.py (hunk 1 above)
Comment (mikeseibel) [inline myuw/dao/affiliation.py]: good catch

**Comment by fanglinfang** [comment]
I approved. but it'd better to have a mock user for verifying the last case.

**Pull request merged**